deploy/operator/ztp/assisted-installer-crds-playbook.yaml

```diff
@@ -30,6 +30,7 @@
     - assisted_stop_after_agent_discovery: "{{ lookup('env', 'ASSISTED_STOP_AFTER_AGENT_DISCOVERY') }}"
     - user_managed_networking: "{{ lookup('env', 'USER_MANAGED_NETWORKING') }}"
     - day2: "false"
+    - baremetalhosts_ignition_override: "{{ lookup('env', 'BAREMETALHOSTS_IGNITION_OVERRIDE', default='False') }}"
```

Comment:
Can we put the override itself inside `BAREMETALHOSTS_IGNITION_OVERRIDE` to make this more flexible?

Reply:
We could, it's just that in o/release we will have the same override multiple times and there is no way of having variables there

Reply:
Create another var, `USE_DEFAULT_DUMMY_BMH_IGNITION_OVERRIDE` and what that is set to true, the script that runs the playbook should set `BAREMETALHOSTS_IGNITION_OVERRIDE` to this dummy value. Then jobs can choose whether they want to just the dummy BMH ignition override or they can set BAREMETALHOSTS_IGNITION_OVERRIDE directly for more flexibility

Reply:
Make s/dummy/arbitrary/ , up to you

Reply:
Nah, this seems to me like overengineering for something we don't really need. Given that for so long now we had no single job testing this, I don't expect suddenly a flood of tests with different overrides. I'll go with the 1st way you suggested and just get content from the variable